assemblies/assembly_business-optimizer-ER-sample-openshift/main.adoc

```diff
@@ -21,6 +21,7 @@ include::{enterprise-dir}/employee-rostering/optashift-ER-overview-con.adoc[leve
 include::{enterprise-dir}/employee-rostering/optashift-ER-openshift-deploy-con.adoc[leveloffset=+1]
 include::{enterprise-dir}/employee-rostering/er-deploy-template-con.adoc[leveloffset=+2]
 include::{enterprise-dir}/openshift/imagestreams-file-install-proc.adoc[leveloffset=+3]
+include::{enterprise-dir}/openshift/secrets-employeerostering-create-proc.adoc[leveloffset=+3]
```

Comment:
Things not in this PR:

* Search "If the application does not open for more than a minute" (In 2.2. DEPLOYING THE APPLICATION USING THE PROVIDED SCRIPT) and note the formatting issues.
* In 3.10. VIEWING EMPLOYEE SHIFTS, the procedure is one step. For single-step procedures, don't number and flush left as a standard sentence/paragraph, per IBM/RH style. That said, I'm not a huge fan of how that procedure is structured, but I don't have the brain power at the moment to review it more carefully and propose a better alternative.
* Something for another day: I'm seeing a lot of steps that start out "To do X, ...", and in multiple sections of the doc, almost all of the steps start that way. That's usually a sign that the procedure isn't really a cohesive procedure for a defined task but a series of options. Best to try to minimize using that, or at least consider if the procedure is really to accomplish a actual task or just a series of tasks. In several cases in this doc, you could just remove that "To do X" bit and slightly reword to make it a genuine step in the procedure.  Just a thought.

Reply:
Created https://issues.jboss.org/browse/BXMSDOC-3906 to track the issues that we don't have the brain power and time for right now.